MOxUnit/moxunit_runtests.m

```diff
@@ -4,22 +4,26 @@
 % result=moxunit_runtests(...)
 %
 % Inputs:
-%   '-verbose'          show verbose output. If not provided, non-verbose
-%                       output is shown.
-%   '-quiet'            do not show output
-%   filename            } test the unit tests in filename
-%   directory           } (which must initialize a test suite through
-%                       } initTestSuite) or in directory. Multiple filename
-%                       or directory arguments can be provided. If there
-%                       are no filename or directory arguments, then tests
-%                       in the current directory are run.
-%   '-logfile', output  store the output in file output. If not provided,
-%                       output id directed to the terminal window
+%   '-verbose'              show verbose output. If not provided,
+%                           non-verbose output is shown.
+%   '-quiet'                do not show output
+%   filename                } test the unit tests in filename
+%   directory               } (which must initialize a test suite through
+%                           } initTestSuite) or in directory. Multiple
+%                           filename or directory arguments can be
+%                           provided. If there are no filename or directory
+%                           arguments, then all tests in the current
+%                           directory are run.
+%   '-logfile', output      store the output in file output. If not
+%                           provided, then output is directed to the
+%                           command window
+%   '-junit_xml', xml    store junit XML output in file xml
 %
 % Output:
-%   result              true if no test failed or raised an error, in other
-%                       words, if all tests either passed or were skipped.
-%                       (therefore, result is true if no tests were run)
+%   result                  true if no test failed or raised an error, in
+%                           words, if all tests either passed or were
```

Comment:
`other` was lost here in rewrapping the text.

Incidentally, this would sound better:
```
+%   result                  true if no test failed or raised an error. In
+%                           other words, true if all tests were either
+%                           successful or skipped.
+%                           (result is true if no tests were run)
```